### DISCUSSION TITLE
Datetime formatting changed from Node v16.16.0 to v16.17.0 but can't find changelog entry

### DISCUSSION THREAD

**Discussion by noahnu** [nodejs/help#4988] [Q&A] [answered]
In node v16.17.0, a change was introduced which causes different datetime formatting when using `Date.prototype.toLocaleDateString`.

In Node v16.16.0, executing:

```js
node -e "console.log(new Date().toLocaleDateString('en-US', {weekday:'long', year:'numeric', month:'long', day: 'numeric', hour: 'numeric', minute: 'numeric'}))"
```

produces:

```
Thursday, October 13, 2022, 10:09 AM
```

while in Node v16.17.0 we get:

```
Thursday, October 13, 2022 at 10:09 AM
```

Note the comma before the timestamp changes to the word "at".

I'm fine with this change, just trying to understand why it was introduced. I can't seem to find any relevant commits or notes in the changelog: https://github.com/nodejs/node/blob/main/doc/changelogs/CHANGELOG_V16.md?plain=1. Could anyone with a better understanding of the node project point me to the commit/issue/discussion?

**Comment by richardlau** [comment, marked as answer]
It's the ICU update (to 71.1). See https://github.com/nodejs/node/issues/44454 for discussion.
